taskcluster/ci/k8s-image/kind.yml

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - tox
+
+transforms:
+    - scriptworker_taskgraph.transforms.python_version:transforms
+    - scriptworker_taskgraph.transforms.k8s_image:transforms
+    - scriptworker_taskgraph.transforms.cached:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    description: "{name} k8s image builder py{python_version}"
+    name: "{name}-python{python_version}"
+    run-on-tasks-for: ["action", "github-pull-request", "github-push"]
+    attributes:
+        code-review: true
+    worker-type: images
+    worker:
+        docker-image: {in-tree: 'k8s-image'}
+        max-run-time: 3600
+        docker-in-docker: true
+        taskcluster-proxy: true
+        chain-of-trust: true
+        artifacts:
+            - type: directory
+              name: public/
+              path: /builds/worker/artifacts/
+        command:
+            - sh
+            - -lxce
+            - >-
+              sh /usr/local/bin/build_and_push.sh
+    python-versions:
+        - 38
+    deploy-secret-url: http://taskcluster/secrets/v1/secret/project/releng/scriptworker-scripts/deploy
+    docker-repo: mozilla/releng-{name}
+    # TODO copy image to artifacts
+
+jobs:
+    addonscript: {}
```

Comment:
Followup: I wonder if we should create a dedicated loader to avoid repeating each subproject here. For instance, there are more than a hundred subprojects all defined [in this file](https://github.com/mozilla-mobile/android-components/blob/6744e62723898db1c5beb24f926303d9279b6030/.buildconfig.yml). [This loader](https://github.com/mozilla-mobile/android-components/blob/6744e62723898db1c5beb24f926303d9279b6030/taskcluster/ac_taskgraph/loader/build_config.py) reads that file, generates the right jobs and applies [the default configuration](https://github.com/mozilla-mobile/android-components/blob/6744e62723898db1c5beb24f926303d9279b6030/taskcluster/ci/build/kind.yml#L20) before passing jobs to transforms.

I see we repeat the list of project here and in another kind so maybe we want a dedicated loader too.

Reply:
Sure, possibly. We could add configuration as to whether we want tox, which versions of python, and whether we want a k8s image (and which versions of python). I think the current breakdown works too.